packages/landing/src/landing.scss

```diff
@@ -384,11 +384,26 @@ header {
     .pt-feature {
       img {
         float: none;
+        margin-right: 0;
         margin-bottom: $pt-grid-size / 2;
         margin-left: -$pt-grid-size / 2;
       }
     }
   }
+
+  .pt-features {
+    .pt-feature-column {
+      width: calc(50% - #{$pt-grid-size * 2});
+
+      &:nth-child(1) {
+        margin-right: $pt-grid-size * 2;
+      }
+
+      &:nth-child(2) {
+        margin-left: $pt-grid-size * 2;
```

Comment:
ok this really looks like it's doing the exact same thing as the original styles above. you've taken 20px off the width and moved it to margin. maybe we can just set `padding` to begin with and not have to change this at all?

these are both two-column layouts, what's actually different about them?